app/controllers/account/dashboard_controller.rb

```diff
@@ -1,8 +1,14 @@
 class Account::DashboardController < ApplicationController
+  LIMIT_TO_SHOW = 5
+
   before_action :authenticate_user!
 
   def index
     @chart_value = current_companies
     @count_of_pages = CompanyPagesAnalytics.count_pages(current_tenant)
+    @recent_companies = current_companies.recent.limit(LIMIT_TO_SHOW)
+    @recent_pages = current_tenant.pages.where(company_id: current_companies.pluck(:id))
```

Comment:
bad code. do without pluck. with joins+where

Reply:
Sorry, I forgot that I have has many through association, so I can call pages directly without 'where'